rust/ql/lib/ext/generated/reqwest/repo-https-github.com-seanmonstar-reqwest-reqwest.model.yml

```diff
@@ -0,0 +1,15 @@
+# THIS FILE IS AN AUTO-GENERATED MODELS AS DATA FILE. DO NOT EDIT.
+extensions:
+  - addsTo:
+      pack: codeql/rust-all
+      extensible: sinkModel
+    data:
+      - ["repo:https://github.com/seanmonstar/reqwest:reqwest", "<crate::async_impl::client::Client>::delete", "Argument[0]", "transmission", "df-generated"]
+      - ["repo:https://github.com/seanmonstar/reqwest:reqwest", "<crate::async_impl::client::Client>::get", "Argument[0]", "transmission", "df-generated"]
+      - ["repo:https://github.com/seanmonstar/reqwest:reqwest", "<crate::async_impl::client::Client>::head", "Argument[0]", "transmission", "df-generated"]
+      - ["repo:https://github.com/seanmonstar/reqwest:reqwest", "<crate::async_impl::client::Client>::patch", "Argument[0]", "transmission", "df-generated"]
+      - ["repo:https://github.com/seanmonstar/reqwest:reqwest", "<crate::async_impl::client::Client>::post", "Argument[0]", "transmission", "df-generated"]
+      - ["repo:https://github.com/seanmonstar/reqwest:reqwest", "<crate::async_impl::client::Client>::put", "Argument[0]", "transmission", "df-generated"]
```

Comment:
This is great, I was going to ask about the lack of [manual] sinks for `get`, `post`, `put`, `patch`, `delete` and `head`!

I wonder what happened to the `blocking::client::Client` versions though.  Perhaps they will come through with a second iteration of the model generator?

Reply:
Yes, good question. I also wondered about that. I think it's because the blocking API is behind and [optional feature flag](https://docs.rs/crate/reqwest/latest/features#blocking) and the extractor uses the default set of features, so the blocking API is not extracted.

Reply:
OK, might be an idea to create an issue about this as it's something we may want to address in the long run?

Reply:
I managed to extract the library with all features enabled (pass `-O cargo_features='*'` when creating the database) and we now get those sink models as well :D